addons/website/static/src/snippets/s_card/001.scss

```diff
@@ -0,0 +1,42 @@
+.s_card[data-vcss="001"] {
+    --card-img-size-h: 33.33333333%;
+    --card-img-ratio-align: 44%;
+    --card-img-aspect-ratio: 20%;
+
+    // Let .card-img inherit the .card's border-radius
+    --card-inner-border-radius: inherit;
+
```

Comment:
I think that we cannot do that 🤔

- if people code  Bootstrap cards without enforcing a background-color, they would get a transparent card 
- If people customize the Bootstrap variable (eg. HTML/CSS Editor - > `user_custom_bootstrap_overridden.scss` -> `$card-bg: red;`) our changes will invalidate the customizations for no good reason 😕

![image](https://github.com/user-attachments/assets/77e0f4ca-08b8-4c60-ab52-1dc490351891)

This look like an editor general bug to me...  otherwise we have to assume that everything could be potentially customized should not have a default white background by default (unless I'm missing something)  🤷🏻‍♂️

Reply:
For me this is not really an issue, any other snippet that has no color is transparent, so it should not be a problem with cards :thinking: In fact it will be consistent with them
Don't forget that this rule is exclusively for the new `s_card` snippet, and this snippet has a default color (the `o_cc1` you added in the template) so every dropped "Card" snippets will be o_cc1 by default :slightly_smiling_face: 
And the user can modify the color of the card by using the option (which should be the main way to change a card color) so there is no need for him to define the color by modifying the BS variable => he should set an inline style in the html editor in that case :thinking: 
I think it is more problematic to not be able to select a color with the option, than to not be able to set a custom variable (since there is the option for that) :slightly_smiling_face: 

When looking at the "Text-Highlight" inner snippet, which has the exact same background options, we see that it doesn't have the issue, because he doesn't have a variable-forced color => so this is really a card-only issue, which is why we think that ignoring the variable is the right solution :slightly_smiling_face: 

But if you are really not sure, we can ignore this for now and ask later if this is the right solution :wink:
Given that your PR is in master, we have time before people start using it :+1:  And the issue will not prevent your PR to be merged, since the templates are correct :wink: (I wanted to find what the issue really was first, because if it was a template issue, it would have impacted everything using the s_card later, but now I am sure the templates are correct :slightly_smiling_face:)

Reply:
> Don't forget that this rule is exclusively for the new s_card snippet

Right, default Bootstrap classes will not be affected indeed, this drastically reduce the issue indeed 👍 

> When looking at the "Text-Highlight" inner snippet, which has the exact same background options, we see that it doesn't have the issue, because he doesn't have a variable-forced color => so this is really a card-only issue,

Not sure because Bootstrap is wide spreading this approach, but we can address it another time indeed 👍 
I'll implement the change, thanks!